notebooks/noxfile_config.py

```diff
@@ -23,7 +23,7 @@
 TEST_CONFIG_OVERRIDE = {
     # You can opt out from the test for specific Python versions.
     # Skipping for Python 3.9 due to pyarrow compilation failure.
-    "ignored_versions": ["2.7", "3.9"],
+    "ignored_versions": ["2.7", "3.6", "3.9"],
```

Comment:
This directory doesn't seem to have any `.py` files, maybe the `noxfile.py` and `noxfile_config.py` should be deleted? https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/notebooks

Reply:
Let's check in with @ivanmkc and/or @loferris who know a ton about the current notebooks testing strategy. My instinct is to say that's probably fine to remove those but I want an LGTM from one of those two folks 😄

Reply:
I'm adding a separate issue to remove so it doesn't block this PR https://github.com/GoogleCloudPlatform/python-docs-samples/issues/7507

Reply:
https://github.com/GoogleCloudPlatform/python-docs-samples/pull/7508